### ISSUE TITLE
Interface improvement: "eye symbol"

### ISSUE THREAD

**Issue by jfcolom** [nicolasbrailo/VlcFreemote#7]
I don't understand the "eye symbol" next to the "plus symbol" in Browse tab. When tapping a media file, it is added to the playlist and the "eye symbol" appears. Then, I remove the media file from the playlist but the "eye symbol" still remains.

Clicking the "eye symbol" makes the app to crash (see stacktrace).

Maybe "eye symbol" could be just removed.

```
09-04 11:10:10.861  5608  5608 E AndroidRuntime: FATAL EXCEPTION: main
09-04 11:10:10.861  5608  5608 E AndroidRuntime: Process: com.nicolasbrailo.vlcfreemote, PID: 5608
09-04 11:10:10.861  5608  5608 E AndroidRuntime: java.lang.RuntimeException: com.nicolasbrailo.vlcfreemote.DirListingView received a click event it can't handle.
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at com.nicolasbrailo.vlcfreemote.DirListingView.onClick(DirListingView.java:133)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at android.view.View.performClick(View.java:5637)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at android.view.View$PerformClick.run(View.java:22433)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:751)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
09-04 11:10:10.861  5608  5608 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

**Comment by nicolasbrailo** [comment]
Thanks for all the feedback! The crash is a regression, I'll fix that ASAP.
The eye symbol means you played that media before. Not very useful for music, a life saver when watching a series of videos. I agree the discoverability of the feature isn't great. Do you have any tips to improve it? I can't think a way to make it more user friendly.

**Comment by jfcolom** [comment]
Ok!

- Maybe long tapping the media item and then tapping "mark as played", or simply "pin", then the "eye symbol" appears.
- Maybe the "eye symbol" is confusing for users who only use VLC for audio. Perhaps something like a [pushpin](https://emojiterra.com/es/chincheta/) is better.
- Maybe this feature should appear in the "Playlist" tab, instead  of the "Browse" tab.

However, I love the simplicity of VlcFreemote, for me it is ok as it is, now I know the meaning.

**Comment by nicolasbrailo** [comment]
Bug fixed, also improved discoverability of view history by adding a short toast message when item is marked as not viewed. Hopefully that makes the intention of this feature clear!

**Comment by jfcolom** [comment]
Thank you, I'm enjoying your fix in VlcFreemote 1.8.

**Comment by nicolasbrailo** [comment]
Happy to hear it, and thanks again for all the valuable feedback!
